.github/pull_request_template.md

```diff
@@ -5,3 +5,10 @@
 **Target Minecraft Versions:** <!-- 'any' means all supported versions -->
 **Requirements:** <!-- Required plugins, Minecraft versions, server software... -->
 **Related Issues:** <!-- Links to related issues -->
+
+<!-- To foster a better open-source community, we ask contributors to release their code under the MIT license -->
+<!-- This license is less restrictive than Skript's current license (GPL) -->
```

Comment:
Isn't it GPL v3?

Reply:
I can specifically cite `GPL v3` there sure